### ISSUE TITLE
setb isn't implemented

### ISSUE THREAD

**Issue by Vurich** [CensoredUsername/dynasm-rs#38]
Currently, dynasm-rs supports setbe, seta, setae, setl, setle, setg, setge, sete and setne but not setb. There's no reason for it not to, it just fails if you try to emit it. As far as I can tell `setc` and `setnae` are precisely equivalent, but it still means that intent is harder to express than it should be. Other synonyms are supported, there's no reason for this one not to be.

**Comment by CensoredUsername** [comment]
I just checked our ops definitions and found this:
```rust
"set
    b"        |
"setc"        |
"setnae"      = [
    b"vb",       [0x0F, 0x92      ], 0;
]
"setn
    b"       |
"setnc"       |
"setae"       = [
    b"vb",       [0x0F, 0x93      ], 0;
]
```

There seems to be an accidental enter in there causing the op to be missing, whoops.

I'll push fixed op definitions, in the meantime you can indeed use the aliases.

**Comment by CensoredUsername** [comment]
This has been fixed in 35456e7 and published in 0.3.0
